### ISSUE TITLE
About lua script

### ISSUE THREAD

**Issue by aspoofer0224** [darksoul577/ReShade#7]
Hi, is it possible to run lua script too ?

**Comment by snajdovski** [comment]
Nope, the SDK is not full, you will need to find patterns and offsets and code that yourself.

**Comment by aspoofer0224** [comment]
With this https://github.com/LWSS/McDota/tree/master/src/SDK, is the sdk enough to run it ? and https://github.com/darksoul577/Simple-Dota-Class-Dumper, i have bunch of lua script, can u please guide me out in discord ?

**Comment by snajdovski** [comment]
> With this https://github.com/LWSS/McDota/tree/master/src/SDK, is the sdk enough to run it ? and https://github.com/darksoul577/Simple-Dota-Class-Dumper, i have bunch of lua script, can u please guide me out in discord ?

It's not that simple, as you think, you will need further patching and reversing, search for an old dota 2 cheat that had doincludescript code in it, its archived on github, good luck! (I don't play or reverse games anymore so you are better off learning a language and reversing by yourself).

**Comment by aspoofer0224** [comment]
> > With this https://github.com/LWSS/McDota/tree/master/src/SDK, is the sdk enough to run it ? and https://github.com/darksoul577/Simple-Dota-Class-Dumper, i have bunch of lua script, can u please guide me out in discord ?
> 
> It's not that simple, as you think, you will need further patching and reversing, search for an old dota 2 cheat that had doincludescript code in it, its archived on github, good luck! (I don't play or reverse games anymore so you are better off learning a language and reversing by yourself).
> Edit 1: Here is the archived cheat:https://github.com/or75/Dota2Hack it has lua execution but you will need to find new offsets and patterns and maybe rewrite some parts.

Thank you

**Comment by AndrewBraunEE** [comment]
> > > With this https://github.com/LWSS/McDota/tree/master/src/SDK, is the sdk enough to run it ? and https://github.com/darksoul577/Simple-Dota-Class-Dumper, i have bunch of lua script, can u please guide me out in discord ?
> > 
> > 
> > It's not that simple, as you think, you will need further patching and reversing, search for an old dota 2 cheat that had doincludescript code in it, its archived on github, good luck! (I don't play or reverse games anymore so you are better off learning a language and reversing by yourself).
> > Edit 1: Here is the archived cheat:https://github.com/or75/Dota2Hack it has lua execution but you will need to find new offsets and patterns and maybe rewrite some parts.
> 
> Thank you

As of November 2020 the IScriptVM has this class structure.

https://www.unknowncheats.me/forum/other-mmorpg-and-strategy/424650-dota-2-iscriptvm.html

I'm currently working on reversing the CDotaPlayer structs (MCDota has some of these already, but they're outdated, and https://github.com/shekeru/dotka-cheats also has some, but it's also outdated).
